source/fundamentals/crud/query-document.txt

```diff
@@ -0,0 +1,331 @@
+
+.. code-block:: go
+   :copyable: false
+   
+   filter := bson.D{{"<field>", bson.D{{"<operator>", "<value>"}}}}
+
+This guide explains how to use :ref:`literal values <go-literal-values>`
+and :ref:`query operators <query-operators-go>` with the ``Find()``
+function.
+
+Sample Data
+~~~~~~~~~~~
+
+To run the examples in this guide, load these documents into the
+``ratings`` collection of the ``tea`` database with the following
+snippet:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CRUD/query.go
+   :language: go
+   :dedent:
+   :start-after: begin insert docs
+   :end-before: end insert docs
+
+.. include:: /includes/fundamentals/automatic-db-coll-creation.rst
+   
+Each document contains a rating for a type of tea and the vendors that
+carry them, which corresponds to the ``rating``, ``type``, and
+``vendor`` fields.
+
+.. include:: /includes/fundamentals/truncated-id.rst
+
+.. _go-literal-values:
+
+Literal Values
+--------------
+
+Literal value query filters return documents with an exact match to your
+match criteria.
+
+.. tip::
+
+   If you specify an empty query filter, CRUD operations match all the
+   documents in a collection.
+
+Example
+~~~~~~~
+
+The following example matches documents where the ``type`` is "Oolong":
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CRUD/query.go
+   :language: go
+   :dedent:
+   :start-after: begin literal
+   :end-before: end literal
+
+After running this example, the output resembles the following:
+
+.. code-block:: none
+   :copyable: false
+
+   [{_id ObjectID("...")} {type Oolong} {rating 7} {vendor [C]}]
+
+.. tip::
+
+   Literal value queries returns the same value as the ``$eq``
+   comparison operator. The following query filters produce the same
+   result:
+
+   .. code-block:: go
+   
+      filter := bson.D{{"type", "Oolong"}}
+
+   .. code-block:: go
+
+      filter := bson.D{{"type", bson.D{{"$eq", "Oolong"}}}}
+
+.. _query-operators-go:
+
+Comparison
+----------
+
+Comparison operators compare the value in a document to the specified
+value in your match criteria. Common comparison operators include
+``$gt`` for "greater than" comparisons, ``$lte`` for "less than or equal
+to" comparisons, and ``$ne`` for "not equal to" comparisons.
+
+Example
+~~~~~~~
+
+The following example matches documents where the ``rating`` is less
+than ``7``:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CRUD/query.go
+   :language: go
+   :dedent:
+   :start-after: begin comparison
+   :end-before: end comparison
+
+After running this example, the output resembles the following:
+
+.. code-block:: none
+   :copyable: false
+
+   [{_id ObjectID("...")} {type English Breakfast} {rating 6}]
+   [{_id ObjectID("...")} {type Assam} {rating 5}]
+
+For a full list of comparison operators, see the :manual:`Comparison
+Query Operators </reference/operator/query-comparison/>` page.
+
+Logical
+-------
+
+Logical operators expect at least 2 match criteria. They check if
```

Comment:
1. Avoid personification. Operators don't really "expect" things.
2. We spell out the numbers one through nine :) [style guide ref](https://docs.mongodb.com/meta/style-guide/style/numbers/#numbers-versus-words)
```suggestion
Logical operators require at least two match criteria. They check if
```